quiz/backend/entrypoint.sh

```diff
@@ -1,14 +1,16 @@
-#!/bin/sh
+#!/bin/bash
 
-if [ "$DATABASE" = "postgres" ]
+set -e
```

Comment:
To w przypadku nieudanej próby połączenia z db, nie wywali nam skryptu?

Reply:
no ale to chcemy, żeby ten skrypt chodził jak nie może się połączyć?

Reply:
to mi zostało przez przypadek po próbach ogarnięcia dla windowsa, sorry

Reply:
Chodzi mi o 9 linijkę, czy tam w warunku jak się nie połączy np. za pierwszym razem to czy set -e nie przerwie całego skryptu. Pytam, bo jeszcze nie jestem bash masterem 😀

Reply:
a bo skomentowałeś trzecią linijkę. Nie jestem bash masterem, ale z tego co rozumiem to netcat nasłuchuje port i jeżeli baza się nie połączy to będzie w nieskończoność jęczeć "Waiting for postgres ... "

Reply:
Bo chodziło o 3 w kontekście 9 😂 ale jak to pozostałość po próbach to nie ma tematu 🙂